src/wallet/wallet.h

```diff
@@ -1129,9 +1129,15 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
     void SetHDChain(const CHDChain& chain, bool memonly);
     const CHDChain& GetHDChain() const { return hdChain; }
 
+    /* Returns true if wallet has an HD seed */
+    bool HasHDSeed() const;
+
     /* Returns true if HD is enabled */
     bool IsHDEnabled() const;
 
+    /* Returns true if the wallet is blank */
```

Comment:
nit: `, i.e. it has no private keys` (because it might have scripts, and the keypool isn't taken into account)